src/components/ordval/guessOption.tsx

```diff
@@ -92,16 +91,38 @@ export const OrdvalGuessOption = ({
       opacity={showSolution ? (solution?.id === guess.id ? 1 : 0.5) : undefined}
       position="relative"
       w="full"
-      h={{ base: "16", md: "32", lg: "40" }}
       {...props}
     >
+      <Image
+        src={guess.pImage ?? guess.image}
+        alt={guess.desc}
+        objectFit="contain"
+        w="full"
+        h={showSolution ? { base: "25vh", md: "72", lg: "96" } : 0}
```

Comment:
Do we need a height conditional here? We've a visibility prop

Reply:
yes, it is a "trick" to load images in the background and show them quickly after

Reply:
Ah, got it. Good one